inc/namespace.php

```diff
@@ -1717,3 +1725,69 @@ function split_large_ep_request( $request, array $query, array $args, int $failu
 
 	return $request;
 }
+
+/**
+ * Handle request forwarding to ES.
+ */
+function handle_autosuggest_endpoint() {
+	if ( '/autosuggest' !== $_SERVER['REQUEST_URI'] ) {
+		return;
+	}
+
+	// Check autosuggest is enabled.
+	$config = Altis\get_config()['modules']['search'];
+	if ( ! ( $config['autosuggest'] ?? false ) ) {
+		return;
+	}
+
+	// Check request is from same origin.
+	$origin = get_http_origin();
+	if ( parse_url( $origin, PHP_URL_HOST ) !== parse_url( get_home_url(), PHP_URL_HOST ) ) {
+		wp_send_json( [], 200 );
+	}
+
+	// Validate data.
+	$json = json_decode( WP_REST_Server::get_raw_data(), true );
+	if ( ! $json ) {
+		wp_send_json( [], 200 );
+	}
+
+	/** @var Features $features Features instance. */
+	$features = Features::factory();
+
+	/** @var Feature\Search\Search $search Search feature instance. */
+	$search = $features->get_registered_feature( 'search' );
+
+	// Force post filter value.
+	$json['post_filter'] = [
+		'bool' => [
+			'must' => [
+				[
+					'term' => [
+						'post_status' => 'publish',
+					],
+				],
+				[
+					'terms' => [
+						'post_type.raw' => array_values( $search->get_searchable_post_types() ),
+					],
+				],
+			],
+		],
+	];
+
+	/** @var Elasticsearch $client */
+	$client = Elasticsearch::factory();
+
+	// Pass to EP.
+	$response = $client->remote_request( ep_get_index_name() . '/post/_search', [
```

Comment:
We should get this directly from the indexable object now instead, `ep_get_index_name()` no longer exists.